plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -255,6 +252,138 @@ p. With this definition we can use expressions like @t.a.x@, see the image below
 p=. Content assist in data structure traversal
 
 
+h2(#cdom_pointers). Pointers
+
+Pointers are a core feature of the C programming language. YAKINDU Statechart Tools' Deep C Integration is making C pointers available to you in your statecharts. In particular, you can
+* declare typed pointer variables,
+* assign a pointer to a pointer variable of the same type,
+* retrieve the pointer pointing to a given variable,
+* pass pointers as parameters to functions, and
+* receive a pointer as a functions return value.
+
+h3(#cdom_declaring-pointer-variables). Declaring pointer variables
+
+Pointer variables are declared in a statechart's definition section as shown in the following example:
+
+bc. var n: int32_t
+var pInt: pointer<int32_t>
+var ppInt: pointer<pointer<int32_t> >
+var pTriangle: pointer<Triangle>;
+
+
+The declarations above declare
+* _n_ as a (non-pointer) variable of type _int32_t_,
+* _pInt_ as a pointer variable that is pointing to a variable of type _int32_t_,
+* _ppInt_ as a pointer that is pointing to a pointer that is pointing to a variable of type _int32_t_, and
+* _pTriangle_ as a pointer to a variable of the self-defined type _Triangle_.
+
+bq.. *Please note:*
+
+When closing the type specification in a pointer declaration with angle brackets, e.&nbsp;g. @pointer<pointer<int32_t> >@, the @>@ characters must be separated from each other by one or more white space characters. Writing e.&nbsp;g. @pointer<pointer<int32_t>>@ would result in an error.
+
+
+h3(#cdom_using-pointer-variables). Using pointer variables
+
+In order to actually assign a pointer to a pointer variable, you have to get hold of that pointer. To retrieve the pointer to a variable _v_, use _v_'s extension function _pointer_. That is, for a variable _v_, the expression _v.pointer_ evaluates to a pointer to _v_. Each variable has the _pointer_ extension function.
+
+Example: Let's say the pointer variable _pInt_ (declared in the example above) should point to the variable _n_. The following assignment accomplishes this:
+
+bc. pInt = n.pointer
+
```

Comment:
Because _v_ is the generic variable and thus represents any and all variables of any and all types and kinds, while _n_ is a concrete variable of type _int32_t_.